common/src/main/java/com/traveltime/plugin/elasticsearch/util/Util.java

```diff
@@ -35,8 +34,8 @@ public static boolean canUseDistance(com.traveltime.sdk.dto.requests.proto.Trans
       return transportation != com.traveltime.sdk.dto.requests.proto.Transportation.Modes.PUBLIC_TRANSPORT;
    }
 
-   public static com.traveltime.sdk.dto.requests.protodistance.Transportation unsafeCastToDistanceTransportation(com.traveltime.sdk.dto.requests.proto.Transportation transportation) {
-      return new com.traveltime.sdk.dto.requests.protodistance.Transportation.Custom(transportation.getValue(), transportation.getCode());
+   public static com.traveltime.sdk.dto.requests.proto.Transportation unsafeCastToDistanceTransportation(com.traveltime.sdk.dto.requests.proto.Transportation transportation) {
+      return new com.traveltime.sdk.dto.requests.proto.Transportation.Custom(transportation.getValue(), transportation.getCode());
```

Comment:
This method is no longer needed.

Reply:
What do you mean it's no longer needed?

Reply:
It was used to cast from `protodistance.Transportation` to `proto.Transportation`. After updating to the newest SDK they're the same type so this can be removed.